configuration/insignia/insignia_colours.sqf

```diff
@@ -0,0 +1,28 @@
+#include "macros.hpp"
```

Comment:
This include isn't needed as this file is included further up the chain into a file that already has this.